src/estimation/include/iDynTree/Estimation/ExtendedKalmanFilter.h

```diff
@@ -254,7 +271,14 @@ namespace iDynTree
          */
         bool ekfGetStateCovariance(const iDynTree::Span<double> &P) const;
 
-    private:
+        /**
+        * function template to ignore unused parameters
+        */
+        template <typename T>
+        void ignore(T &&)
+        { }
+
+            private:
```

Comment:
Shouldn't this be inline with the `public` keywork that is already in the class?

Reply:
inline you mean in the same intendation?

Reply:
Yes sorry, I actually mean `in line`, not `inline` . : )